src/main/java/net/mcreator/ui/minecraft/villagers/JVillagerTradeProfession.java

```diff
@@ -0,0 +1,135 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.villagers;
+
+import net.mcreator.element.parts.VillagerTradeEntry;
+import net.mcreator.element.types.VillagerTrade;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.help.IHelpContext;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.ui.minecraft.JEntriesList;
+import net.mcreator.workspace.Workspace;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class JVillagerTradeProfession extends JEntriesList {
+
+	private final JComboBox<String> villager = new JComboBox<>();
+
+	private final List<JVillagerTradeEntry> entryList = new ArrayList<>();
+
+	private final Workspace workspace;
+
+	private final JPanel entries = new JPanel(new GridLayout(0, 1, 5, 5));
+
+	public JVillagerTradeProfession(MCreator mcreator, IHelpContext gui, JPanel parent, List<JVillagerTradeProfession> professionList) {
+		super(mcreator, new BorderLayout(), gui);
+
+		setOpaque(false);
+
+		this.workspace = mcreator.getWorkspace();
+
+		final JComponent container = PanelUtils.expandHorizontally(this);
+
+		parent.add(container);
+		professionList.add(this);
+
+		ElementUtil.loadAllVillagerProfessions(workspace).forEach(e -> villager.addItem(e.getName()));
+
+		setBackground(((Color) UIManager.get("MCreatorLAF.DARK_ACCENT")).brighter());
+
+		JPanel topbar = new JPanel(new FlowLayout(FlowLayout.LEFT));
+		topbar.setOpaque(false);
+
+		topbar.add(L10N.label("elementgui.villager_trade.profession"));
+		topbar.add(villager);
+
+		topbar.add(Box.createHorizontalGlue());
+
+		JButton add = new JButton(UIRES.get("16px.add.gif"));
+		add.setText(L10N.t("elementgui.villager_trade.add_entry"));
+
+		JButton remove = new JButton(UIRES.get("16px.clear"));
+		remove.setText(L10N.t("elementgui.villager_trade.remove_profession_trades"));
+		remove.addActionListener(e -> {
+			professionList.remove(this);
+			parent.remove(container);
+			parent.revalidate();
+			parent.repaint();
+		});
+
+		JComponent component = PanelUtils.centerAndEastElement(topbar, PanelUtils.join(FlowLayout.RIGHT, add, remove));
+		component.setOpaque(true);
+		component.setBackground(((Color) UIManager.get("MCreatorLAF.DARK_ACCENT")).brighter());
+
+		add("North", component);
+		entries.setOpaque(false);
+
+		add.addActionListener(e -> {
+			JVillagerTradeEntry entry = new JVillagerTradeEntry(mcreator, gui, entries, entryList);
+			registerEntryUI(entry);
+		});
+		add("Center", entries);
+
+		setBorder(BorderFactory.createTitledBorder(
+				BorderFactory.createLineBorder((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"), 2),
```

Comment:
Use border thickness 1, we are moving to lighter borders

Reply:
Done!